added github from scratch. added philipp to authors. refreshed dianas description

fabiansch opened the pull request.

fabiansch committed:
added github from scratch to blog. added philipp to authors. refreshed diana's discription

gnunicorn commented on _posts/2013-12-03-githubfromscratch-hamburg-december.md:
```diff
@@ -0,0 +1,39 @@
+authors:
+  - diana
+  - philipp
+image: /assets/content/2013-12-03-githubfromscratch-hamburg-december/0.jpeg
```
I can't find this or any of the other mentioned pictures. Did you forget to add them?

fabiansch replied:
oh yes. Sorry. Give me a few minutes ;)

Am 04.12.2013 um 10:59 schrieb Benjamin Kampmann:

> In _posts/2013-12-03-githubfromscratch-hamburg-december.md:
> 
> > +---
> > +published: true
> > +layout: post
> > +title: "GitHub from Scratch - „I fork you and you fork me“"
> > +teaser: "OTS Hamburg has created a new workshop: Github from Scratch"
> > +description: "A great one day workshop teaching Github from Scratch"
> > +category:
> > +tags:
> > -  - github
> > -  - workshop
> > -  - hamburg
> > -  - beginners
> >   +authors:
> > -  - diana
> > -  - philipp
> >   +image: /assets/content/2013-12-03-githubfromscratch-hamburg-december/0.jpeg
> >   I can't find this or any of the other mentioned pictures. Did you forget to add them?
> 
> —
> Reply to this email directly or view it on GitHub.

gnunicorn commented on _posts/2013-12-03-githubfromscratch-hamburg-december.md:
```diff
@@ -0,0 +1,39 @@
+published: true
+layout: post
+title: "GitHub from Scratch - „I fork you and you fork me“"
+teaser: "OTS Hamburg has created a new workshop: Github from Scratch"
```
This is the snippet we show on the index/overview page. I'd write a little more here to get people on the page wanting to read the article. In doubt, we usually just copy-paste the first paragraph here.

fabiansch replied:
ah super. Okay :)

fabiansch committed:
added pictures for new blogpost github from scratch

gnunicorn commented:
@fabiansch no reason to close. if you just push to the same branch all future changes become part of the same pull request.

gnunicorn commented:
@fabiansch if you'd now add a bit more of a teaser, I'm happy to merge it.

fabiansch committed:
modified teaser

fabiansch committed:
bugfix on teasermessage

fabiansch committed:
updated date

fabiansch commented:
Done :)

fabiansch committed:
Delete 2013-12-03-githubfromscratch-hamburg-december.md

deleted this file because I updated the date of the blogpost and this needs to rename the file..
